tests/testthat/test-target_market_share.R

```diff
@@ -521,3 +521,24 @@ test_that("w/ technology in ald but not loanbook, outputs all techs (#235)", {
   expect_equal(corporate_economy$ice$technology_share, 0.25)
   expect_equal(corporate_economy$electric$technology_share, 0.75)
 })
+
+test_that("w/ unweighted company flags & multi loans, outputs correctly (#239)", {
+  matched <- fake_matched(id_loan = c("L1", "L2"))
+
+  ald <- fake_ald()
+
+  scenario <- fake_scenario()
```

Comment:
Nit: I think these three statements belong to the same "initialization paragraph" (a common pattern in tests is to divde the code in three paragraphs -- initialization, computation, expectation). I would use not white line between them.

Reply:
sounds good and agreed! do you have a resource that explains the pattern of test division?

Reply:
Pretty sure it's in "Clean code". I have a hard copy here ... I'll see if I can find it.